### DISCUSSION TITLE
My set thumbnail button isn't working. Is there any way to fix it?

### DISCUSSION THREAD

**Discussion by VietnameseCoder** [ScratchAddons/ScratchAddons#5650] [Help]
When I set a thumbnail, it always doesn't show anything and when I close and go to the projects section the thumbnail is unchanged. Scratch Addons team, can you help me with this?

**Comment by WorldLanguages** [comment]
It might take some time for the thumbnail to update. Try checking if it updated from a different device or browser.
Also, make sure you're not saving the project after setting the thumbnail.

**Reply by VietnameseCoder** [reply]
Ok then!

**Reply by VietnameseCoder** [reply]
nope, that's not what happened.

**Comment by DNin01** [comment]
When you aren't using this addon, do the thumbnails update or no?

**Reply by VietnameseCoder** [reply]
no, they don't

**Reply by DNin01** [reply]
It might not be caused by Scratch Addons, then.

**Reply by VietnameseCoder** [reply]
Might be true..... Haven't updated chrome in a while....

**Comment by DNin01** [comment]
I probably should have asked if disabling the entire extension allowed it to save. If you disable Scratch Addons and any other browser extensions you're using then save the project, does the thumbnail update?

**Reply by WorldLanguages** [reply]
@LeVuong2013 You should have clarified what you meant by "When I set a thumbnail". Do you mean the "set thumbnail" feature from Scratch Addons? Or do you mean saving the project normally?
